drivers/SmartThings/zigbee-switch/src/aqara/multi-switch/init.lua

```diff
@@ -0,0 +1,120 @@
+local device_lib = require "st.device"
+local capabilities = require "st.capabilities"
+local clusters = require "st.zigbee.zcl.clusters"
+local cluster_base = require "st.zigbee.cluster_base"
+local data_types = require "st.zigbee.data_types"
+local constants = require "st.zigbee.constants"
+
+local SimpleMetering = clusters.SimpleMetering
+
+local PRIVATE_CLUSTER_ID = 0xFCC0
+local PRIVATE_ATTRIBUTE_ID = 0x0009
+local MFG_CODE = 0x115F
+
+local FINGERPRINTS = {
+  { model = "lumi.switch.n1acn1", children = 1, child_profile = "" },
+  { model = "lumi.switch.n2acn1", children = 2, child_profile = "aqara-switch-child" },
+  { model = "lumi.switch.n3acn1", children = 3, child_profile = "aqara-switch-child" },
+}
+
+local function do_configure(self, device)
+  if device.network_type == device_lib.NETWORK_TYPE_ZIGBEE then
+    device:configure()
+    device:send(cluster_base.read_manufacturer_specific_attribute(device,
+      PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID, MFG_CODE))
+    device:send(SimpleMetering.attributes.CurrentSummationDelivered:configure_reporting(device, 900, 3600, 1)) -- minimal interval : 15min
```

Comment:
Set the minimum interval allowed between reports of SimpleMetering to 15 minutes like the reporting interval of powerConsumptionReport.